recipes/pybda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pybda" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 541908e3874550de3fdceb118279e7530180e1fe9caf7319fe0e29408227f788
+
+build:
+  number: 0
+  skip: true # [not py35 and not py37]
```

Comment:
Aren't you going for `skip: True  # [not py36]` ? You can probably just remove this, make it `noarch: python` and then specify `python 3.6` in the host and run sections.

Reply:
This is actually causing it to not build anything, which is why all of the testing passes (there's nothing to fail).

Reply:
Oh man, thanks!